rewrite-core/src/main/java/org/openrewrite/Tree.java

```diff
@@ -36,6 +38,10 @@ default String getJacksonPolymorphicTypeTag() {
         return getClass().getName();
     }
 
+    default IndentType getIndentType() {
+        return IndentType.NONE;
+    }
+
```

Comment:
As explained, this is not how continuations work. Depending on where it appears a method invocation's prefix may be a continuation or it might not be. So it cannot be a property of the tree itself.

Reply:
Yes, I understand that, this is a default indent type, and it can be overridden by code depending on where it appears.
because we are using a way of putting messages on a cursor to let the `autoFormattor` know the indent type like this 
```java
getCursor().putMessage("indentType", IndentType.CONTINUATION_INDENT);
```
currently, we have default indentType for each element as well (in `preVisit` as a central place), and it will be overridden by the algorithm of formatters if it depends on the appearance place, and context. 

the benefit of this is it will allow elements like `K.Return` to tell what is the default indent type of it. otherwise from AutoFormat perspective, it has no idea about the correct indent type.